test/html/HtmlReplaceMakerTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+require_once 'src/wovnio/html/HtmlReplaceMarker.php';
+
+use Wovnio\Html\HtmlReplaceMarker;
+
+class HtmlReplaceMarkerTest extends PHPUnit_Framework_TestCase {
```

Comment:
Please make tests with at least 11 keys.
I don't see the problem here, but first wovnphp version had revert issues with keys ending by `1` and `11` (or `2` and `20`, etc.). We need to secure that case for future changes.

Reply:
done!

Reply:
thanks